config.yaml

```diff
@@ -177,6 +177,15 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-1.9.3.3
```

Comment:
There is a flaw in a recent commit (a096800) which lead to duplication of links under the wrong label.

The correct name of the package for OpenMage Magento mirror is ` name: magento-mirror-1.9.3.3`, please move it to the right place.